spec/pluralrules.html

```diff
@@ -46,20 +46,18 @@
     </emu-clause>
 
     <emu-clause id="sec-GetOperands" aoid="GetOperands">
-       <h1>GetOperands (x)</h1>
+       <h1>GetOperands (s)</h1>
        <p>
-         When the GetOperands abstract operation is called with argument _n_ (which must be a finite Number value), it returns an Object value with the list of plural operands used by plural rules functions.
+         When the GetOperands abstract operation is called with argument _s_ (which must be a String matching the regular expression pattern `-?\d*(\.\d*)?`, it returns an Object value with the list of plural operands used by plural rules functions.
        </p>
 
        <p>
         The following steps are taken:
        </p>
 
        <emu-alg>
+         1. Let _n_ be ? ToNumber(_s_).
```

Comment:
this is not needed, the caller of this abstract operation should provide a number, otherwise the assertion will throw (which means ecmascript is broken).